web3c/index.js

```diff
@@ -1,5 +1,11 @@
 /* globals Web3 */
-const MraeBox = require('../crypto/subtle/mrae_box');
+let MraeBox = undefined;
```

Comment:
this means you can also remove the alias from webpack?

Reply:
(and in fact may need to or the node instance here will still get re-written, no?)